src/js/components/Drop/stories/Custom.js

```diff
@@ -33,7 +34,8 @@ const CustomDrop = () => {
         </Box>
         {targetRef.current && (
           <Drop
-            align={{ top: 'bottom', left: 'right' }}
+            elevation="xsmall"
```

Comment:
Do we need to change this story? Or can we leave as it was

Reply:
we can leave as is